Sources/XCUITestHelpers/XCUIElement+Async.swift

```diff
@@ -0,0 +1,21 @@
+import XCTest
+
+extension XCUIElement {
+
+    /// Beware that this uses `XCTNSPredicateExpectation` which relies on polling and is slower than
+    /// other asynchronous expectations.
+    @discardableResult
+    public func waitForIsHittable(timeout: TimeInterval = 3.0) -> Bool {
```

Comment:
The name in the code I extracted this from was `waitForHittability`, which I guess was inspired from the native `waitForExistence` API. My grammar checker warns on "hittability" so I went for "is hittable" which is correct English even thought the name doesn't read as smoothly as "wait for existence".

Keen to hear if you have other naming suggestions.

Reply:
I agree that `waitForHittability` is more pleasant for the eye, but `waitForIsHittable` describes what the method does more precisely, so I'm for keeping it the way it is.